app/client/src/components/shared/Histogram.tsx

```diff
@@ -89,6 +88,23 @@ export default function Histogram({
             },
           },
         },
+        menuItemDefinitions: {
+          printChart: {
+            onclick: function () {
+              const divToPrint = document.getElementById(uuid);
```

Comment:
The chart ref would be used here instead:
```suggestion
              const divToPrint = chartRef.current?.container.current;
```